src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -560,6 +564,36 @@ def getSystemParameters(self):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def ledOn(self):
```

Comment:
`ledOn()` and `ledOff()` nearly do the same except instruction code. Use one function with boolean parameter to turn LED on or off.